package.json

```diff
@@ -5,41 +5,32 @@
     "license": "CC-BY-3.0",
     "repository": "https://github.com/fluid-project/infusion-docs",
     "scripts": {
-        "docpad": "node node_modules/docpad/bin/docpad run --env static --out out",
-        "deploy": "node node_modules/docpad/bin/docpad deploy-ghpages --env static",
-        "pretest": "node node_modules/docpad/bin/docpad generate --env static --out out",
+        "11ty": "eleventy --serve",
+        "build": "eleventy",
         "test": "npm run test:node && npm run test:browser",
         "test:node": "node tests/all-node-tests.js",
         "test:browser": "node node_modules/testem/testem.js ci --file tests/testem.js"
     },
     "dependencies": {
-        "docpad": "6.79.4",
-        "docpad-plugin-ghpages": "2.6.1",
-        "docpad-plugin-handlebars": "2.4.0",
-        "docpad-plugin-highlightjs": "2.5.0",
-        "docpad-plugin-markit": "2.4.0",
-        "foundation-sites": "5.5.2",
-        "fs-extra": "4.0.2",
-        "gpii-binder": "1.0.5",
-        "gpii-location-bar-relay": "1.0.4",
-        "infusion": "3.0.0-dev.20190328T144119Z.ec44dbfab",
-        "lunr": "2.3.6",
-        "markdown-it-anchor": "4.0.0",
-        "mkdirp": "0.5.1",
-        "normalize.css": "3.0.1",
-        "octicons": "3.1.0",
-        "urijs": "1.19.0"
+        "@11ty/eleventy": "^0.11.0",
```

Comment:
The versions were previously (and intentionally) all pinned, i.e. they did not use the leading caret to indicate that later versions were acceptable.  I would suggest removing the carets.

Reply:
Done.